From e67810006b7adf120bbb2f6c0c84697ec4b783e5 Mon Sep 17 00:00:00 2001
From: Benjamin Franzke <benjaminfranzke@googlemail.com>
Date: Fri, 10 Feb 2012 00:56:44 +0100
Subject: [PATCH] logind: Allow PowerOff/Reboot in default context

(cherry picked from commit 80df5f4c3b697d666324fc37e63661f44a5f76cf)

Conflicts:

	src/org.freedesktop.login1.conf
---
 src/org.freedesktop.login1.conf |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/src/org.freedesktop.login1.conf b/src/org.freedesktop.login1.conf
index c423ef5..96b4ae3 100644
--- a/src/org.freedesktop.login1.conf
+++ b/src/org.freedesktop.login1.conf
@@ -69,6 +69,14 @@
                        send_member="ActivateSession"/>
 
                 <allow send_destination="org.freedesktop.login1"
+                       send_interface="org.freedesktop.login1.Manager"
+                       send_member="PowerOff"/>
+
+                <allow send_destination="org.freedesktop.login1"
+                       send_interface="org.freedesktop.login1.Manager"
+                       send_member="Reboot"/>
+
+                <allow send_destination="org.freedesktop.login1"
                        send_interface="org.freedesktop.login1.Seat"
                        send_member="ActivateSession"/>
 
